packages/manager/modules/pci/src/projects/project/instances/instance/detach-private-network/detach-private-network.routing.js

```diff
@@ -0,0 +1,29 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'pci.projects.project.instances.instance.detachPrivateNetwork',
+    {
+      url: '/network/private/detach',
+      views: {
+        modal: {
+          component: 'pciInstancesInstanceDetachPrivateNetwork',
+        },
+      },
+      layout: 'modal',
+      resolve: {
+        goBack: /* @ngInject */ (goToInstance) => {
+          return goToInstance;
+        },
+        privateNetworks: /* @ngInject */ (
```

Comment:
Is this call needed ?

Reply:
The goBack() one ?

Reply:
yes, it it needed in ordre to dismiss the popup.